src/main.h

```diff
@@ -2,25 +2,25 @@
 #define MAIN_H
 
 #define MAIN_TRACE_FILTER (0 \
-        | LSHIFT(ERROR) \
-        | LSHIFT(WARN) \
-        | LSHIFT(DEBUG) \
-        | LSHIFT(INFO) \
-        | LSHIFT(INIT) \
+        | BITLSHIFT(1, ERROR) \
+        | BITLSHIFT(1, WARN) \
+        | BITLSHIFT(1, DEBUG) \
+        | BITLSHIFT(1, INFO) \
+        | BITLSHIFT(1, INIT) \
         )
 
 #if 1
-extern const char *main_trace_header[];
+// extern const char *main_trace_header[];

```

Comment:
Remove unnecessary code/comment like main_trace_header.

Reply:
OK

Reply:
I had just fixed it and push the new commit.